src/sentry/utils/hashlib.py

```diff
@@ -33,7 +33,7 @@ def hash_value(h, value):
             hash_value(h, item)
     elif isinstance(value, dict):
         h.update(b"\x05" + str(len(value)).encode("utf-8"))
-        for k, v in value.items():
+        for k, v in sorted(value.items()):
```

Comment:
Fixing a iteration order bug in `hash_values` that we would encounter while hashing tag values. Nobody else other than us is passing dictionaries into this function so this is not breaking any other user.